module2/gift-certificate-system/web/src/test/java/com/epam/esm/validator/TagValidatorTest.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm.validator;
+
+import com.epam.esm.model.Tag;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.validation.BeanPropertyBindingResult;
+import org.springframework.validation.BindingResult;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class TagValidatorTest {
+
+    private TagValidator validator;
+
+    @BeforeEach
+    void setUp() {
+        validator = new TagValidator();
+    }
+
+    @Test
+    void validate() {
```

Comment:
Please check best practices for writing unit tests, read also for FIRST acronym. I can see several concerns here:
1. Naming is to general. When such method fails it is not clear what was wrong with test.
2. Single responsibility principle. Your test contains two scenarios. Move second part to separate method, for example 
```
@Test
public void shouldThrowErrorForLongName()
```
2. In first part you create valid tag object but there is no any assertion. So if this logic does not work the test will pass without any error.

Reply:
Fixed